src/gr_sparse_mat/test/main.c

```diff
@@ -0,0 +1,27 @@
+#include <string.h>
```

Comment:
A lot of copyright claims are missing. It is useful to know who wrote the code, so one knows who to contact if something needs to be fixed.